### DISCUSSION TITLE
Is there a simple way to toggle the react-dnd drag sorting functionality?

### DISCUSSION THREAD

**Discussion by drewgallagher** [ant-design/ant-design#43569] [Q&A]
I am trying to build out a feature that allows users to drag columns on their own while still being able to drag rows. 

In order to do this, it seems like I need to toggle on and off the react-dnd behavior so it doesn't override the column selection. 

When I tried swapping out the components object it causes major performance issues because it rerenders the whole table as a new table every time that is swapped out based on a state variable. 

Is there another way to disable it? Maybe a css class or something? 

I also tried setting the canDrag property with a custom prop, but it seems that prop cannot be changed because it gets managed inside the Table component

here is the implementation I am following https://4x.ant.design/components/table/#components-table-demo-drag-sorting

**Comment by linxianxi** [comment]
can you try v5 drag demo? it used `dnd-kit`, it's better then `react-dnd`.
